src/_posts/platform/app/2000-01-01-send-signal.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Send a signal to a container
+nav: Send Signal
+modified_at: 2023-01-12 00:00:00
+tags: app signal
+---
+
+## Introduction
+
+Signals are standardized messages sent to a running program to trigger specific behavior.
+
+On the Scalingo platform, sending signals is a useful way to trigger events while being the owner or collaborator of an application. This can be particularly useful for debugging purposes.
+
+{% note %}
+  The platform handles only `SIGUSR1` and `SIGUSR2`.
+{% endnote %}
+
+## General purpose of the feature
+
+As said before, implementing a signal handler can be an easy and powerful way to debug applications.
```

Comment:
Either you already said it and you don't need to say it again, or you think it's really important to have this sentence at the beginning of this section and you don't need to mention that you already said it.

```suggestion
```